hs_core/page_processors.py

```diff
@@ -324,7 +327,8 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
                'extended_metadata_layout': extended_metadata_layout,
                'bag_url': bag_url,
                'show_content_files': show_content_files,
```

Comment:
Should `show_content_files` reflect the discoverable/public flags of the resource?